README.md

```diff
@@ -62,12 +62,12 @@ If you choose to enable X11 support:
 * xcb
 * xcb-composite
 * xcb-xfixes
+* xcb-xinput
```

Comment:
(This could be marked as optional, only needed for the X11 backend)

Reply:
All of the xcb* dependencies could, because Xwayland is optional too.

Reply:
Right